internal/output/json.go

```diff
@@ -92,8 +99,22 @@ func (f *JSON) Error(value interface{}) {
 
 	_, err = f.cfg.OutWriter.Write(b)
 	if err != nil {
-		multilog.Error("Could not write json output, error: %v", err)
+		if isPipeClosedError(err) {
+			logging.Error("Could not write json output, error: %v", err) // do not log to rollbar
+		} else {
+			multilog.Error("Could not write json output, error: %v", err)
+		}
+	}
+}
+
+func isPipeClosedError(err error) bool {
+	pipeErr := errors.Is(err, syscall.EPIPE)
+	if runtime.GOOS == "windows" && errors.Is(err, syscall.Errno(242)) {
```

Comment:
As long as the comment is correct which I think it is.
```suggestion
	if runtime.GOOS == "windows" && errors.Is(err, syscall.Errno(232)) {
```

Reply:
🤦 Thanks